apps/web/components/sections/project/success-gallery.tsx

```
@@ -0,0 +1,39 @@
+      <div className="flex items-center justify-between mb-4">
+        <h3 className="text-lg font-bold mb-4">Success Gallery</h3>
+        <button className="text-sm font-medium text-blue-600 hover:underline">View All</button>
+      </div>
+      <div className="grid grid-cols-1 sm:grid-cols-3 gap-4">
+        {items.map((item) => (
+          <div
+            key={item.src} 
+            className="w-full h-40 bg-gray-100 rounded-lg overflow-hidden shadow-sm cursor-pointer"
+            onClick={() => onMediaClick?.(items.indexOf(item))} 
+            onKeyDown={(e) => e.key === 'Enter' && onMediaClick?.(items.indexOf(item))} 
+            role="button" 
```

Biome (1.9.4)

[error] The elements with the following roles can be changed to the following elements:
<button>
<input type="button">


For examples and more information, see WAI-ARIA Roles

(lint/a11y/useSemanticElements)